src/lib/assetBridger/ethBridger.ts

```diff
@@ -344,18 +304,75 @@ export class EthBridger extends AssetBridger<
     await this.checkL1Network(params.l1Signer)
     await this.checkL2Network(params.l2Provider)
 
-    const retryableTicketRequest = isL1ToL2TransactionRequest(params)
-      ? params
-      : await this.getDepositToRequest({
-          ...params,
-          from: await params.l1Signer.getAddress(),
-          l1Provider: params.l1Signer.provider!,
-        })
+    const l1Provider = params.l1Signer.provider
 
-    const tx = await params.l1Signer.sendTransaction({
-      ...retryableTicketRequest.txRequest,
-      ...params.overrides,
-    })
+    if (!l1Provider) {
+      throw new Error('l1Provider not found')
+    }
+
+    const inbox = Inbox__factory.connect(
+      this.l2Network.ethBridge.inbox,
+      params.l2Provider
+    )
+
+    const signerAddress = new Address(await params.l1Signer.getAddress())
+    const signerAliasedAddress = signerAddress.applyAlias()
+
+    const l1ToL2MessageGasEstimator = new L1ToL2MessageGasEstimator(
+      params.l2Provider
+    )
+
+    const destinationAddress = isL1ToL2TransactionRequest(params)
+      ? params.txRequest.to
+      : params.destinationAddress
+
+    const amount = isL1ToL2TransactionRequest(params)
+      ? BigNumber.from(params.txRequest.value)
+      : params.amount
+
+    const gasOverrides = isL1ToL2TransactionRequest(params)
+      ? {
+          gasLimit: {
+            base: params.overrides?.gasLimit
+              ? BigNumber.from(params.overrides.gasLimit)
+              : undefined,
+          },
+          maxFeePerGas: {
+            base: params.overrides?.maxFeePerGas
+              ? BigNumber.from(params.overrides.maxFeePerGas)
+              : undefined,
+          },
+        }
+      : params.retryableGasOverrides
+
+    const gasEstimation = await l1ToL2MessageGasEstimator.estimateAll(
+      {
+        from: signerAliasedAddress.value,
+        to: destinationAddress,
+        l2CallValue: amount,
+        excessFeeRefundAddress: destinationAddress,
+        callValueRefundAddress: destinationAddress,
+        data: '0x',
+      },
+      await getBaseFee(l1Provider),
+      l1Provider,
+      gasOverrides
+    )
+
+    const tx = await inbox.connect(params.l1Signer).createRetryableTicket(
+      destinationAddress, // to,
+      amount, // l2CallValue
+      gasEstimation.maxSubmissionCost, // maxSubmissionCost
+      destinationAddress, // excessFeeRefundAddress
+      destinationAddress, // callValueRefundAddress
+      gasEstimation.gasLimit, // maxLimit
+      gasEstimation.maxFeePerGas, // maxFeePerGas
+      '0x', // data,
+      {
+        from: signerAddress.value,
+        value: amount,
```

Comment:
actually this should be
```suggestion
        value: gasEstimation.deposit,
```